src/MainWindow/main_window.cpp

```diff
@@ -195,6 +195,19 @@ void MainWindow::createMenus() {
 
   helpMenu = menuBar()->addMenu("&Help");
   helpMenu->addAction(aboutAction);
+
+  m_progressWidget = new QWidget;
```

Comment:
Any reason to not create a separate function for creating this wedget ? e.g., createProgressWidget()

Reply:
Done